### ISSUE TITLE
launch init throwing 'Error: ENOENT: no such file or directory ~/.npm-packages/lib/node_modules/meteor-launch/assets/launch.json'

### ISSUE THREAD

**Issue by steveovens** [NewSpring/meteor-launch#93]
Ran the npm install and tried 'launch init' and got this error:

```
~/src/paintball/app [dev*]$ launch init
local@xxx-Macbook-Pro~$ 
fs.js:549
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/xxx/.npm-packages/lib/node_modules/meteor-launch/assets/launch.json'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at fs.readFileSync (fs.js:397:15)
    at /Users/xxx/.npm-packages/lib/node_modules/meteor-launch/lib/util.js:77:45
    at FSReqWrap.oncomplete (fs.js:82:15)
```

I was able to resolve this by going to my node_modules/meteor-launch directory and creating a new 'assets' folder and copying the launch.json file from github into it.

After doing this, `launch init` worked fine and created the launch.json file in my app.

**Comment by johnthepink** [comment]
@steveovens sorry about that. I must have done a bad publish somewhere. I just published version 0.3.2, can you try that and see if it resolves the issue for you? Thanks!

**Comment by johnthepink** [comment]
@steveovens hope this is working for you now. If not, feel free to reopen. Thanks!
